### PR TITLE
adjust plot titles

### PR DESCRIPTION
**Pull request by Rensvandeschoot**

### CODE REVIEW

File: asreviewcontrib/insights/plot.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def _plot_recall(ax, labels, x_absolute=False, y_absolute=False):
         yticks = [0, 0.2, 0.4, 0.6, 0.8, 1.0]
 
     ax.step(x, y, where='post')
-    ax.set_title("Recall")
+    ax.set_title("Recall plot")
```
Comment (J535D165) [inline asreviewcontrib/insights/plot.py:72]: Remove plot? Or use plot with the others as well? I prefer the first.

File: asreviewcontrib/insights/plot.py [hunk 2]
```diff
@@ -186,7 +186,7 @@ def _plot_wss(ax, labels, x_absolute=False, y_absolute=False):
         yticks = [0, 0.2, 0.4, 0.6, 0.8, 1.0]
 
     ax.step(x, y, where='post')
-    ax.set_title("WSS")
+    ax.set_title("Work Saved over Sampling (WSS) versus Recall")
```
Comment (J535D165) [inline asreviewcontrib/insights/plot.py:189]:
```suggestion
    ax.set_title("Work Saved over Sampling (WSS) given Recall")
```

What is preferred? Ignore if disagree.

File: asreviewcontrib/insights/plot.py (hunk 1 above)
Comment (Rensvandeschoot) [reply asreviewcontrib/insights/plot.py:72]: Recall is already used on the y-axis. So, what is the title of this graph?

Comment (J535D165) [reply asreviewcontrib/insights/plot.py:72]: The same holds for ERF. We can add the project name to it later.

Comment (Rensvandeschoot) [reply asreviewcontrib/insights/plot.py:72]: Alternative: Recall/erf at every step of the active learning process

Comment (Rensvandeschoot) [inline asreviewcontrib/insights/plot.py:72]:
```suggestion
    ax.set_title("Relevant records found during the screening phase")
```

Comment (Rensvandeschoot) [reply asreviewcontrib/insights/plot.py:72]: see new suggestion below

Comment (J535D165) [inline asreviewcontrib/insights/plot.py:72]:
```suggestion
    ax.set_title("Recall")
```

**Pull request merged**